scripts/formal/fpv.tcl

```diff
@@ -0,0 +1,23 @@
+// Copyright 2024 Cirrus Logic
+// SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
+
+set DESIGN_RTL_DIR ../../rtl
+
+analyze -sv -f ../../cv32e40p_fpu_manifest.flist
+analyze -sva -f cv32e40p_formal.flist
```

Comment:
Is it cv32e40p_formal.flist or tb_src/cv32e40p_tb_src.flist?

I don't find first one anywhere nor auto-generated anywhere.
Originally we used it to list all bind, assume and assertion files.

Reply:
cv32e40p_formal.flist is already part of the repo over here --> cv32e40p/scripts/formal/cv32e40p_formal.flist
It's not a new file I have added, I have just reused for jasper.